src/providers/index.ts

```diff
@@ -144,20 +144,10 @@ export async function generateCompletions (force = false, project: Project): Pro
 		}
 		const sdkPath = sdkInfo.path;
 		// Generate the completions
-		const [ alloy, sdk ] = await Promise.all([
+		await Promise.all([
 			completion.generateAlloyCompletions(force, CompletionsFormat),
 			completion.generateSDKCompletions(force, sdkVersion, sdkPath, CompletionsFormat)
 		]);
-		if (sdk || alloy) {
```

Comment:
I realised this is pretty useless information to a developer so just removed the info message